### PR TITLE
test(jepsen): run Jepsen using SCT

### PR DESCRIPTION
**Pull request by enaydanov**
Trello: https://trello.com/c/uuQuikM5/2360-jepsen
Staging job: https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/evgeniy/job/jepsen/

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [x] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [x] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [x] All new and existing unit tests passed (CI)
- [x] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: vars/jepsenPipeline.groovy [hunk 1]
```diff
@@ -0,0 +1,141 @@
+#! groovy
```
Comment (bentsi) [inline vars/jepsenPipeline.groovy:1]: use general `vars/runSendEmail.groovy`, `vars/runCollectLogs.groovy`, `vars/runCleanupResource.groovy`

File: test-cases/jepsen/jepsen.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+gce_root_disk_type_db: 'pd-ssd'
+gce_root_disk_size_db: 50
+gce_n_local_ssd_disk_db: 1
+instance_provision: "on_demand"
+n_db_nodes: 5
+n_loaders: 1
+n_monitor_nodes: 1
+nemesis_class_name: 'NoOpMonkey'
+scylla_linux_distro: 'debian-buster'
+store_results_in_elasticsearch: False
```
Comment (bentsi) [inline test-cases/jepsen/jepsen.yaml:17]: false

File: test-cases/jepsen/jepsen.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+cluster_backend: 'gce'
+gce_image_username: 'sct'
+gce_image_db: 'https://www.googleapis.com/compute/v1/projects/debian-cloud/global/images/family/debian-10'
+gce_image_loader: 'https://www.googleapis.com/compute/v1/projects/debian-cloud/global/images/family/debian-10'
+gce_instance_type_db: 'n1-standard-4'
+gce_instance_type_loader: 'n1-highmem-8'
+gce_root_disk_type_db: 'pd-ssd'
+gce_root_disk_size_db: 50
+gce_n_local_ssd_disk_db: 1
+instance_provision: "on_demand"
```
Comment (bentsi) [inline test-cases/jepsen/jepsen.yaml:11]: remove, add to job

File: test-cases/jepsen/jepsen.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+backtrace_decoding: false
```
Comment (bentsi) [inline test-cases/jepsen/jepsen.yaml:1]: remove

File: test-cases/jepsen/jepsen.yaml [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+gce_root_disk_size_db: 50
+gce_n_local_ssd_disk_db: 1
+instance_provision: "on_demand"
+n_db_nodes: 5
+n_loaders: 1
+n_monitor_nodes: 1
+nemesis_class_name: 'NoOpMonkey'
+scylla_linux_distro: 'debian-buster'
+store_results_in_elasticsearch: False
+test_duration: 360
```
Comment (bentsi) [inline test-cases/jepsen/jepsen.yaml:18]: move to the beginning

**Review by bentsi — CHANGES_REQUESTED** [review]
please address comments

File: vars/jepsenPipeline.groovy (hunk 1 above)
Comment (enaydanov) [reply vars/jepsenPipeline.groovy]: done

File: test-cases/jepsen/jepsen.yaml (hunk 2 above)
Comment (enaydanov) [reply test-cases/jepsen/jepsen.yaml]: done

File: test-cases/jepsen/jepsen.yaml (hunk 3 above)
Comment (enaydanov) [reply test-cases/jepsen/jepsen.yaml]: done

File: test-cases/jepsen/jepsen.yaml (hunk 4 above)
Comment (enaydanov) [reply test-cases/jepsen/jepsen.yaml]: done

File: test-cases/jepsen/jepsen.yaml (hunk 5 above)
Comment (enaydanov) [reply test-cases/jepsen/jepsen.yaml]: done

**Review by bentsi — APPROVED** [review]

**Pull request merged**